src/SleepLog.js

```diff
@@ -1,22 +1,15 @@
 class SleepLog {
 
   constructor(sleepData, userID) {
-    this.log = sleepData.filter((data) => data.userID === userID)
-      .map(userLog => {
-        const logEntry = {};
-        logEntry.date = userLog.date;
-        logEntry.hoursSlept = userLog.hoursSlept;
-        logEntry.sleepQuality = userLog.sleepQuality;
-        return logEntry;
-      });
+    this.log = sleepData.filter((data) => data.userID === userID);
   }
 
   calculateAllTimeAverageSleep(variable) {
     let totalSleep = 0;
-    log.forEach((entry) => {
+    this.log.forEach(entry => {
       totalSleep += entry[variable];
     });
-    return totalSleep / entry.length;
+    return (totalSleep / this.log.length).toFixed(1);
```

Comment:
I added this `.toFixed(1)`. You might know what this is but JIC, it works kinda like `Math.floor()` but it rounds to the amount of decimals that you pass in as an argument. So something like 8.564720 would be rounded to 8.6.

Here's an [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed) on it!

Reply:
Sweet!